api/v1alpha1/batchrelease_plan_types.go

```diff
@@ -50,6 +50,10 @@ type ReleasePlan struct {
 	// FinalizingPolicy define the behavior of controller when phase enter Finalizing
 	// Defaults to "Immediate"
 	FinalizingPolicy FinalizingPolicyType `json:"finalizingPolicy,omitempty"`
+	// PatchPodTemplateMetadata indicates patch configuration(e.g. labels, annotations) to the canary deployment podTemplateSpec.metadata
+	// only support for canary deployment
```

Comment:
is support for cloneset possible?

Reply:
This still depends on the scenario, there is no canary workload for cloneSet, so it may be necessary to look at the scenario later